text/2019-01-08-pd-simulator.md

```diff
@@ -0,0 +1,150 @@
+# PD Simulator
+
+## Summary
+
+This RFC proposes a new tool named **Simulator** in PD. It is used to simulate
+different cases in real scenarios in order to evaluate the correctness and
+performance of scheduling of PD.
+
+## Motivation
+
+Currently, it's a little bit hard to tell if the PD has some scheduling problems
```

Comment:
```suggestion
Currently, it's a little bit hard to tell if PD has some scheduling problems
```

@CaitinChen do you think we should drop the "the" here?

Reply:
@huachaohuang Yes. Drop "the", please.